lib/offline/storage.js

```diff
@@ -770,6 +788,61 @@ shaka.offline.Storage.prototype.createOfflineManifest_ = function(
   };
 };
 
+/**
+ * Extract DRM initData from first segment
+ *
+ * @param {shaka.extern.Manifest} manifest
+ * @param {shaka.net.NetworkingEngine} networkingEngine
+ * @return {Promise<shaka.extern.Manifest>}
+ * @private
+ */
+shaka.offline.Storage.prototype.extractInitDataFromFirstSegment_ = async function(
+  manifest, networkingEngine) {
+  // get init segments uri of all variants
+  /** @type {Array<Array<string>>} */
+  let initSegmentReferences = [].concat.apply([], manifest.periods.map((period) => {
```

Comment:
Our coding standards have been updated that all variables that are only set once must be declared with "const". Please update your code to reflect that.

Reply:
we can update to the latest shaka standards